docs/docs-content/cluster-templates/create-cluster-template-policies/maintenance-policy.md

```diff
@@ -0,0 +1,193 @@
+---
+sidebar_label: "Maintenance Policies"
+title: "Maintenance Policies"
+description: "Learn what maintenance policies are, including how to create, edit, and delete them."
+hide_table_of_contents: false
+sidebar_position: 10
+tags: ["cluster templates", "policies"]
+---
+
+:::preview
+
+:::
+
+A maintenance policy is a required component for [cluster templates](../cluster-templates.md). Maintenance policies
+determine when and how upgrades are executed on the cluster. Since clusters can be provisioned across multiple regions
+and time zones, upgrades are executed based on Coordinated Universal Time (UTC).
+
+When the cluster profile version linked to a cluster template is updated, or when profiles are added, deleted, or
+replaced, the associated maintenance policy automatically schedules upgrades for all active clusters. These clusters are
+then upgraded during the upgrade window defined in the maintenance policy. Each maintenance policy can have multiple
+upgrade schedules, and upgrades can be [manually triggered](../modify-cluster-templates.md#overview-tab) outside of the
+defined schedule, giving you the flexibility to trigger upgrades across clusters whenever necessary.
+
+All clusters must be on the same version and all cluster profile variables must be in an **Assigned** state before
+upgrading to the next version. If some clusters fail to upgrade during the upgrade window, or if clusters deployed with
```

Comment:
```suggestion
All clusters must be on the same version and all cluster profile variables must be in an **Assigned** state before
upgrading to the next version.
```

I don't understand that sentence. What are we trying to say here?

Reply:
```suggestion
All clusters must be on the same cluster profile version, and all cluster profile variables must be in an **Assigned**
state before they can be upgraded to the next cluster profile version. If some clusters fail to upgrade during the upgrade window, or if clusters deployed with
```